libshaderc/include/shaderc.h

```diff
@@ -113,6 +113,13 @@ void shaderc_compile_options_release(shaderc_compile_options_t options);
 void shaderc_compile_options_add_macro_definition(
     shaderc_compile_options_t options, const char* name, const char* value);
 
+// Set the compiler to only do preprocessing. When both preprocessing only mode
+// and disassembly mode are turned on, the compiler will respect preprocessing
+// only
```

Comment:
Bad line break in comment.
Preprocessing-only mode also prevents SPIR-V code generation.  You should generalize the comment that way.
E.g.
"
Set the compiler to do only preprocessing.  The byte array result in the module returned by the compilation method is the text of the preprocessed shader."

Also, you now must update the descriptions on the methods: shaderc_compile_into_spv, shaderc_module_get_bytes, shaderc_module_get_length.  They all say the result is a SPIR-V binary. But that's no longer true.